eks-worker-al2-variables.json

```diff
@@ -13,7 +13,7 @@
     "binary_bucket_region": "us-west-2",
     "cache_container_images": "false",    
     "cni_plugin_version": "v0.8.6",
-    "containerd_version": "1.6.6-1.amzn2.0.2",
+    "containerd_version": "1.6.*",
```

Comment:
Do we need to make any changes in: 
- MCM template https://mcm.amazon.com/cms/MCM-75796329
- AMI changelog
to acomodate this?

We hardcode containerd version in both the above places.